lib/cfme/cloud_services/payload_builder.rb

```diff
@@ -0,0 +1,107 @@
+require 'minitar'
+
+module Cfme
+  module CloudServices
+    module PayloadBuilder
+      BOUNDARY = "AaB03x".freeze
+
+      module RequestParams
+        UPLOADER_HOST = "localhost".freeze
+        UPLOADER_PORT = "8080".freeze
+        UPLOADER_URI = "/api/ingress/v1/upload".freeze
+
+        CONTENT_TYPE = "application/vnd.redhat.topological-inventory.something+tgz".freeze
+
+        def org_id
+          54_321 # TODO: tie with subscription
+        end
+
+        def tenant
+          12_345 # TODO: tie with subscription
+        end
+
+        def x_rh_insights_request_id
+          # TODO: generate?
+          # request_id: Unique ID provided to the payload created by 3Scale.
+          # This ID will be used for the life of the object.
+          "52df9f748eabcfea"
```

Comment:
I _think_ this is actually generated by 3scale and not something that we have to pass

Reply:
I am not sure about at this moment and I need to learn about it more - I guess we need to we need to obtain request_id from 3scale first ? if so I guess it can be done follow up PR.

Reply:
this not needed now

Reply:
:+1: